packages/server/src/db/index.js

```diff
@@ -460,6 +486,27 @@ async function getAgencies(rootAgency) {
     return result.rows;
 }
 
+async function getAllAgencies() {
```

Comment:
This accessor should take a `tenantId` param and add a where clause to the query based on it. Then caller in `/allagencies/` endpoint (perhaps we should rename?) should filter based on `req.session.user.tenant_id`.